package.json

```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "dev": "next & json-server --watch -p 4000 db.json",
     "start": "next start",
-    "test": "jest"
+    "test": "jest --updateSnapshot",
```

Comment:
You should't add the `--updateSnapshot` on the `test` command. This way it will always update the snapshots.

The `npm run test` command is used in the build for checking if the tests succeed. If you add the flag `--updateSnapshot` to that command your snapshots tests will never fail because it will create a new snapshots. 

The way snapshots work: 
If you run a jest test for a component, it will create a snapshot file. The next time you will run jest it will compare the old snapshot with a new one. If both are same your test will succeed. It will fail if the snapshots are not the same, in this case you can take action (for example change something in component).

There are basically 2 scenario's for a snapshot failure:
1: You implemented something new (consciously)
2: You implemented something new (accidentally)

- If you added a new feature and you are sure that it works properly you can update the snapshot. So next time it will compare changes with your new snapshot.
- In case of something that you implemented accidentally (regression) you probably won't need to update the snapshot. Just fix the component and run "jest" again until it works again.

I would create a new command:
```
"test": "jest",
"test-watch": "jest --watch",
"test-update":  "jest --updateSnapshot"
```

Now you can manually run updateSnapshot when you need to.

Reply:
Ok thanks!